pyccel/ast/class_defs.py

```diff
@@ -157,6 +157,8 @@
                 decorators = {}),
             PyccelFunctionDef('clear', func_class = SetClear,
                 decorators={}),
+            PyccelFunctionDef('copy', func_class = SetCopy,
+                decorators={}),
```

Comment:
`{}` is the default argument for `decorators`. Is there any reason this argument is being passed explicitly every time?

Reply:
You're right, I don't see any benefits of calling an empty decorator. I will remove them in my next commit.

Reply:
Fixed : 66fe285